src/ResourceManager/AzureBackup/Commands.AzureBackup.Test/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Commands.AzureBackup.Test")]
```

Comment:
be consistent, please reference the same constants, such as 
https://github.com/Azure/azure-powershell/blob/dev/src/ServiceManagement/Storage/Commands.Storage.Test/Properties/AssemblyInfo.cs#L22